src/nvim/ex_cmds2.c

```diff
@@ -641,11 +645,11 @@ void ex_breaklist(exarg_T *eap)
  * Find a breakpoint for a function or sourced file.
  * Returns line number at which to break; zero when no matching breakpoint.
  */
-linenr_T 
-dbg_find_breakpoint (
-    int file,                   /* TRUE for a file, FALSE for a function */
-    char_u *fname,         /* file or function name */
-    linenr_T after             /* after this line number */
+linenr_T
+dbg_find_breakpoint(
+    int file,                   // TRUE for a file, FALSE for a function
+    char_u *fname,              // file or function name
+    linenr_T after              // after this line number
```

Comment:
This was to silent clint, it was warning that old style-comments are not allowed, and function name should not be followed by a space before parenthesis. Travis build was failing otherwise.